accounts-db/src/transaction_results.rs

```diff
@@ -1,3 +1,5 @@
+// Re-exported since these have moved to `solana_sdk`.
+pub use solana_sdk::inner_instruction::{InnerInstruction, InnerInstructionsList};
```

Comment:
Can you please add a deprecated attribute to this? It won't actually warn callers, but at least it's a nice code annotation for when we clean these up.

Reply:
Sure, as of which version? `1.18.0`?

Reply:
Yep, v1.18.0